### PR TITLE
Added sort by function in task list

### PR DESCRIPTION
**Pull request by mlataza**
- Added sort by feature in TaskList component
- Added sortByProperty in utils folder

### CODE REVIEW

File: src/routes/Dashboard/TaskList.js [hunk 1]
```diff
@@ -8,35 +8,88 @@ import TableRow from "@mui/material/TableRow";
 import Paper from "@mui/material/Paper";
 import { Link, Button } from "@mui/material";
 
-import TextField from "@mui/material/TextField";
+//import TextField from "@mui/material/TextField";
```
Comment (bbynosa) [inline src/routes/Dashboard/TaskList.js]: Can you please remove commented statement

File: src/routes/Dashboard/TaskList.js [hunk 2]
```diff
@@ -8,35 +8,88 @@ import TableRow from "@mui/material/TableRow";
...
 import {
   Select,
   MenuItem,
+  FormControl,
+  InputLabel
 } from "@mui/material";
-import { v4 as uuidv4 } from "uuid";
+//import { v4 as uuidv4 } from "uuid";
```
Comment (bbynosa) [inline src/routes/Dashboard/TaskList.js]: Can you please remove commented statement

File: src/routes/Dashboard/TaskList.js (hunk 1 above)
Comment (mlataza) [reply src/routes/Dashboard/TaskList.js]: Got it.

File: src/routes/Dashboard/TaskList.js (hunk 2 above)
Comment (mlataza) [reply src/routes/Dashboard/TaskList.js]: Got it.

**Review by bbynosa — APPROVED** [review]
lgtm!

**Pull request merged**